server/server.js

```diff
@@ -221,10 +221,21 @@ app.post('/flic_button_press', Validator.header(['button-serial-number']).exists
         if(validationErrors.isEmpty()){
             const serialNumber = req.get('button-serial-number')
             const batteryLevel = req.get('button-battery-level')
+            const buttonName = req.get('button-name')
+            const apiKey = req.query.apikey
+
+            // Log the vaiditiy of the API key
+            // TODO (CU-gwxnde) Replace this with a 401 unauthorized if invalid
```

Comment:
the client that would be receiving this is the Flichub itself right? So right now it can't do anything with a 401 if it did receive it? (Or I guess a spoofer would also be receiving this if they tried to spoof without an API key?)

Reply:
and to be clear - right now we're not rejecting any requests with this? Just logging?

Reply:
Yes, yes, yes, and yes :smile:

Reply:
And yes. (I think I got all the question marks now, hehe).

I'm just logging for now because we can't actually reject requests until every button is set up with the key.